buildimage.go

```diff
@@ -65,7 +72,7 @@ func main() {
 			"-files", extraBinMust("wpa_action"),
 			"-files", extraBinMust("wpa_cli"),
 			"-files", extraBinMust("wpa_passphrase"),
-			"-files", "webboot/webboot:bbin/webboot")
+			"-files", filepath.Join(currentDir+"/webboot/webboot")+":bbin/webboot")
```

Comment:
filepath.Join(currentDir, "webboot", webboot")
is technically best: will work on winDoze should we ever go crazy.